src/builder/cfg.rs

```diff
@@ -245,7 +245,7 @@ pub type BlockBuilder<B> = DFGWrapper<B, BasicBlockID>;
 
 impl<B: AsMut<Hugr> + AsRef<Hugr>> BlockBuilder<B> {
     /// Set the outputs of the block, with `branch_wire` being the value of the
-    /// predicate.  `outputs` are the remaining outputs.
+    /// Choice.  `outputs` are the remaining outputs.
```

Comment:
Why is it `branch_wire` not `choice_wire` (why wasn't it `predicate_wire` before)?